modules/http2/h2_workers.h

```diff
@@ -38,19 +38,17 @@ struct h2_workers {
     
     int next_worker_id;
     apr_uint32_t max_workers;
-    volatile apr_uint32_t min_workers; /* is changed during graceful shutdown */
-    volatile apr_interval_time_t max_idle_duration; /* is changed during graceful shutdown */
-    
-    volatile int aborted;
-    volatile int shutdown;
+    apr_uint32_t min_workers;
+    apr_uint32_t worker_count;
+    apr_uint32_t max_idle_secs;
+    apr_uint32_t aborted;
+    apr_uint32_t shutdown;
```

Comment:
Ditto for volatile "documentation" (for those which are still accessed atomically)

Reply:
ack.